charts/datadog/templates/_container-cri-volumemounts.yaml

```diff
@@ -0,0 +1,16 @@
+{{- define "container-crisocket-volumemounts" -}}
+{{- if eq .Values.targetSystem "linux" }}
+- name: runtimesocketdir
+  mountPath: {{ print "/host/" (dir (include "datadog.dockerOrCriSocketPath" .)) | clean }}
+  mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
+  readOnly: true
+{{- end }}
+{{- if eq .Values.targetSystem "windows" }}
+- name: runtimesocket
+  mountPath: {{ template "datadog.dockerOrCriSocketPath" . }}
+{{- if not .Values.datadog.criSocketPath }}
+- name: containerdsocket 
+  mountPath: \\.\pipe\containerd-containerd
```

Comment:
I think that in other yamls I needed to use `` like `\\.\pipe\containerd-containerd` to avoid escaping.

Reply:
IMO it was because you used the env vars to set it in your test.

```console
$ kubectl get pod -o yaml <pod-name>Volumes:
Volumes:
  // ...
  runtimesocket:
    Type:          HostPath (bare host directory volume)
    Path:          \\.\pipe\containerd-containerd
    HostPathType:
```